DataProfileViewer/_plot_metadata_table.py

```diff
@@ -38,61 +34,61 @@ def id_generator(size=15):
     """Helper function to generate random div ids. This is useful for embedding
     HTML into ipython notebooks."""
     chars = list(string.ascii_uppercase)
-    return ''.join(np.random.choice(chars, size, replace=True))
+    return ''.join(random.choices(chars, k=size))
```

Comment:
This is the only use of `numpy`, so I thought I'd remove it, even though datamart-profiler depends on numpy anyway.